homeassistant/components/trello/const.py

```diff
@@ -0,0 +1,14 @@
+"""Constants for the trello integration."""
+import logging
+from typing import Final
+
+DOMAIN: Final = "trello"
+
+LOGGER = logging.getLogger(__package__)
+
+CONF_API_TOKEN = "api_token"
+CONF_USER_ID = "user_id"
+CONF_USER_EMAIL = "user_email"
+CONF_BOARD_IDS = "board_ids"
+
+CONF_OPTIONS_BOARDS = "boards"
```

Comment:
CONF and OPTIONS sound a bit the same, how does CONF_BOARDS sound?

Reply:
I think I was trying to differentiate keys for options and config data, but not sure there's any good reason to do that. So I think this change would make sense.